### ISSUE TITLE
RuntimeError: wrapped C/C++ object of type QLineEdit has been deleted

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1455]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/home/ago/src/Electron-Cash/plugins/shuffle/qt.py", line 1698, in <lambda>
    QTimer.singleShot(100, lambda: onFormChange())
  File "/home/ago/src/Electron-Cash/plugins/shuffle/qt.py", line 1693, in onFormChange
    d = self.get_form()
  File "/home/ago/src/Electron-Cash/plugins/shuffle/qt.py", line 1501, in get_form
    'server': self.le.text(),

RuntimeError: wrapped C/C++ object of type QLineEdit has been deleted
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.7  | 3.7.3 (default, Mar 26 2019, 21:43:19)  [GCC 8.2.1 20181127] | Linux-5.1.9-arch1-1-ARCH-x86_64-with-arch | standard | en_US | 

Additional Information
------------------------

> Axel here. Opened CashShuffle settings and pressed esc really quickly

---

**Comment by cculianu** [comment]
deleteLater ( ͡° ͜ʖ ͡°)

**Comment by cculianu** [comment]
Fixed in latest commit:  c3b4a2cb8022baf3170e47b7f5eaeca5beac7589
